pages/common/cpio.md

```diff
@@ -1,16 +1,15 @@
 # cpio
 
-> Archiving utility.
+> Copies files to and from archives.
```

Comment:
The description from https://www.gnu.org/software/cpio/ is quite readable and explains the acronym :)

> cpio copies files into or out of a cpio or tar archive